rrmngmnt/filesystem.py

```diff
@@ -89,3 +89,39 @@ def create_script(self, content, path):
                 raise errors.CommandExecutionFailure(
                     executor, cmd, rc, err,
                 )
+
+    def wget(self, url, f_dir):
+            """
+            Download file on the host from given url
+
+            :param url: url to file
+            :type url: str
+            :param f_dir: file directory on host
+            :type f_dir: str
+            :return: absolute path to file
+            :rtype: str
+            """
+            rc = None
+            host_executor = self.host.executor()
+            file_path = os.path.join(f_dir, url.split('/')[-1])
```

Comment:
you can have url like: http://something/getfile.php?name=myfile&format=pdf

I would prefer change f_dir to output_file and you can skip this logic.